### PR TITLE
[libpq] Disable thread safety check under thread sanitizer

### PR DESCRIPTION
**Pull request by mmatrosov**
Specify library name and version:  **libpq/13.3**

Workaround for this problem: https://www.postgresql.org/message-id/1513971675.5870501.1439797066345.JavaMail.yahoo@mail.yahoo.com

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libpq/all/patches/13.3/001-Disable-thread-safety-check-under-thread-sanitizer.patch [hunk 1]
```diff
@@ -0,0 +1,19 @@
+diff --git a/src/test/thread/thread_test.c b/src/test/thread/thread_test.c
+index e1bec01..e4ffd78 100644
+--- a/src/test/thread/thread_test.c
```
Comment (madebr) [inline recipes/libpq/all/patches/13.3/001-Disable-thread-safety-check-under-thread-sanitizer.patch:3]: This is a patch for a test.
Does libpq build/run tests?

Comment (mmatrosov) [reply recipes/libpq/all/patches/13.3/001-Disable-thread-safety-check-under-thread-sanitizer.patch:3]: This test is run during the configure phase. This is not exactly a test, this is more of a feature check.

**Comment by mmatrosov** [comment]
There is some discussion going on the mailing list of the library: https://www.postgresql.org/message-id/CADhDkKzPSiNvA3Hyq%2BwSR_icuPmazG0cFe%3DYnC3U-CFcYLc8Xw%40mail.gmail.com

**Comment by mmatrosov** [comment]
@jgsogo the build service is running for more than 20 hours now, is it expected? Could you please suggest how long does it usually take?

**Comment by SSE4** [comment]
> @jgsogo the build service is running for more than 20 hours now, is it expected? Could you please suggest how long does it usually take?

he is on vacation right now. CI build queue is around ~500 jobs now, with pretty complex packages like boost. unfortunately, CI resources are limited. even if we do most of the things in parallel, unfortunately there are still numerous bottlenecks which make job allocation far from the ideal. of course there is a room for improvement.
how long does it usually take really depends on the number of active PRs. right now we have many boost PRs running in parallel, plus several smaller libraries. I'd say it may take several hours more.

**Review by madebr — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Comment by mmatrosov** [comment]
It's been 10 days since the PR was merged, but I don't see a new recipe revision in conan-center:
```
$ conan search -r conan-center libpq/13.3@ -rev
Revisions for 'libpq/13.3' at remote 'conan-center':
43ae0b4a8b396d05adcec63c28db1d12 (2021-06-22 17:15:17 UTC)
```
Is there something I have to do to publish the new revision?

**Comment by madebr** [comment]
I have:
```
$ conan search -r conan-center libpq/13.3@ -rev
Revisions for 'libpq/13.3' at remote 'conan-center':
a4861fe4293a56eb62cbcacb8195a954 (2021-07-26 13:36:53 UTC)
64bcc1a2c134d7418f2b72aba9c1382f (2021-06-28 14:34:58 UTC)
43ae0b4a8b396d05adcec63c28db1d12 (2021-06-22 17:15:17 UTC)
```

**Comment by jgsogo** [comment]
> It's been 10 days since the PR was merged, but I don't see the new recipe revision in conan-center:
> 
> ```
> $ conan search -r conan-center libpq/13.3@ -rev
> Revisions for 'libpq/13.3' at remote 'conan-center':
> 43ae0b4a8b396d05adcec63c28db1d12 (2021-06-22 17:15:17 UTC)
> ```
> 
> Is there something I have to do to publish the new revision?

Hi, @mmatrosov , please read here about the Conan remotes: https://blog.conan.io/2021/06/10/New-conan-release-1-37.html. The remote at URL `https://conan.bintray.com` has been frozen for some months now (no packages are being uploaded) and we are uploading packages to `https://center.conan.io`. Conan releases starting in 1.37 already add this new remote.

Please, check your remotes with `conan remote list` and add the new center if needed, probably that's the issue.

**Comment by mmatrosov** [comment]
@jgsogo yes, that's exactly the issue, thanks!

**Pull request merged**